Process.c

```diff
@@ -782,19 +782,11 @@ void Process_writeField(const Process* this, RichString* str, ProcessField field
       }
 
       char* buf = buffer;
-      int maxIndent = 0;
-      bool lastItem = (this->indent < 0);
-      int indent = (this->indent < 0 ? -this->indent : this->indent);
+      const bool lastItem = (this->indent < 0);
 
-      for (int i = 0; i < 32; i++) {
-         if (indent & (1U << i)) {
-            maxIndent = i + 1;
-         }
-      }
-
-      for (int i = 0; i < maxIndent - 1; i++) {
+      for (int indent = (this->indent < 0 ? -this->indent : this->indent); indent > 1; indent >>= 1) {
```

Comment:
Do we have some reason to avoid `int indent = abs(this->indent)`?

Reply:
I don't see one …

Reply:
With the latest push `this->indent` is of type `int64_t`, which would depending on the system either require `labs(3)` or `llabs(3)`.
Also the manpage states:

> Trying to take the absolute value of the most negative integer is not defined.
> ...
> By default, GCC handles abs(), labs(), and (since GCC 3.0) llabs() and imaxabs() as built-in functions.

So I do not see any benefit, as modern compilers should recognize the pattern `a < 0 ? -a : a`.